src/test/integration/integration-tests.py

```diff
@@ -0,0 +1,167 @@
+        # Create test event
+        valid_event = utils.post_event(event_body)
+        valid_id = valid_event.json()["data"]["id"]
+
+        # Invalid audienceIDs
+        validate_bad_list(self, "audienceIDs", valid_id)
+        # Invalid countyIDs
+        validate_bad_list(self, "countyIDs", valid_id)
+        # Invalid departmentIDs
+        validate_bad_list(self, "departmentIDs", valid_id)
+        # Invalid eventTopicIDs
+        validate_bad_list(self, "eventTopicIDs", valid_id)
+        # Invalid eventTypeIDs
+        validate_bad_list(self, "eventTypeIDs", valid_id)
+
+        # Clean up test event
+        utils.delete_event(valid_id)
+
+    # Test GET /calendar/event-types and GET /calendar/event-types/{id}
+    def test_event_types(self):
+        test_get(self, "event-types")
+
+    # Test GET /calendar/event-topics and GET /calendar/event-topics/{id}
+    def test_event_topics(self):
+        test_get(self, "event-topics")
+
+    # Test GET /calendar/locations and GET /calendar/locations/{id}
+    def test_locations(self):
+        test_get(self, "locations")
+
+    # Test GET /calendar/counties and GET /calendar/counties/{id}
+    def test_counties(self):
+        test_get(self, "counties")
+
+    # Test GET /calendar/campuses and GET /calendar/campuses/{id}
+    def test_campuses(self):
+        test_get(self, "campuses")
+
+    # Test GET /calendar/departments and GET /calendar/departments/{id}
+    def test_departments(self):
+        test_get(self, "departments")
+
+    # Test GET /calendar/audiences and GET /calendar/audiences/{id}
+    def test_audiences(self):
+        test_get(self, "audiences")
+
+
+def validate_response(self, response, code=None, res_type=None, message=None):
+    if code:
+        self.assertEqual(response.status_code, code)
+    if res_type:
+        self.assertEqual(response.json()["data"]["type"], res_type)
+    if message:
+        self.assertIn(message, response.json()[0]["developerMessage"])
+
+
+def validate_bad_list(self, list_name, put_id=None):
+    event_body["data"]["attributes"][list_name] = []
+    event_body["data"]["attributes"][list_name].append("badItem")
+    if put_id:
+        bad_response = utils.put_event(put_id, event_body)
+    else:
+        bad_response = utils.post_event(event_body)
+    validate_response(self, bad_response, 400, message=list_name)
+    event_body["data"]["attributes"][list_name] = None
+
+
+def test_get(self, result):
+    # Test GET /{result}
+    valid_results = utils.get_results(result)
+    validate_response(self, valid_results, 200)
+
+    # Test GET /{result}/{id} with valid ID
+    try:
+        valid_id = valid_results.json()["data"][0]["id"]
+        valid_result = utils.get_result_by_id(result, valid_id)
+        validate_response(self, valid_result, 200, result)
+    # No results in GET /result so GET /result/{id} can't be
+    # tested with a valid ID
+    except IndexError:
+        print("Can't test GET /calendar/{}/{{id}} with valid ID. "
```

Comment:
Could you use a logger instead of print? example: https://github.com/osu-mist/hr-api/blob/44a379c769e6c598b75313e21335853a04c22160/src/test/integration/integration_tests.py#L63